### PR TITLE
Add Notes in Create Pipeline Using Jenkinsfile/Editing Panels-v3.1

### PR DESCRIPTION
**Pull request by Felixnoo**
Signed-off-by: Felixnoo <felixliu@yunify.com>

### CODE REVIEW

**Comment by Felixnoo** [comment]
@kubesphere/sig-devops Please help review the updates I made in this PR when you are available, thanks.

File: content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-graphical-editing-panel.md [hunk 1]
```diff
@@ -96,17 +96,31 @@ You can use the account `project-admin` to create the project. Besides, this acc
 4. The pipeline created will appear in the list.
 
    ![pipeline-list](/images/docs/devops-user-guide/using-devops/create-a-pipeline-using-graphical-editing-panels/pipeline-list.jpg)
+   
+   {{< notice note >}}
+   
+   The **Sync Status** colume shows the synchronization result between KubeSphere and Jenkins, and you can see the icon of **Success** if the synchronization is successful.
+   
+   {{</ notice >}}
```
Comment (LinuxSuRen) [inline content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-graphical-editing-panel.md:104]: Please hold on about this part. We're still under discussion about this. See also https://github.com/kubesphere/kubesphere/issues/3639

File: content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-jenkinsfile.md [hunk 2]
```diff
@@ -239,6 +245,12 @@ The account `project-admin` needs to be created in advance since it is the revie
    input(id: 'release-image-with-tag', message: 'release image with tag?', submitter: 'project-admin,project-admin1')
    ···
    ```
+   
+   {{< notice note >}}
+   
+   In KubeSphere v3.1, only pipeline creator, account with the role of admin in the project, or the account you specify will have the authority to continue or terminate the pipeline.
```
Comment (LinuxSuRen) [inline content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-jenkinsfile.md:251]: There're two cases here:

* First, anyone who can run this pipeline, then he can approve or reject it if there are no specific approvers
* Second, same as you described above

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]

File: content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-graphical-editing-panel.md (hunk 1 above)
Comment (Felixnoo) [reply content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-graphical-editing-panel.md]: I'll remove this part in the PR.

File: content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-jenkinsfile.md (hunk 2 above)
Comment (Felixnoo) [reply content/en/docs/devops-user-guide/how-to-use/create-a-pipeline-using-jenkinsfile.md]: Thanks you the comment and I made some changes.

**Comment by Sherlock113** [comment]
@LinuxSuRen Hi, do you have any comments about this pr?

**Comment by Felixnoo** [comment]
@LinuxSuRen Hi, please take a look at this PR when you are available.

**Review by LinuxSuRen — APPROVED** [review]
/lgtm

**Pull request merged**